File: api/session/status.ts
// Vercel zero-config serverless function → GET /api/session/status
// Server-authoritative free-session state (quota, expiry, provider/model).
import { sessionStatusHandler } from '../../src/api/handlers.ts';
import { withHttp } from '../../src/api/middleware.ts';

export default withHttp(sessionStatusHandler);

File: api/session/status.py
"""Vercel zero-config Python function → GET /api/session/status.

Server-authoritative free-session state (quota, expiry, provider/model).
"""

import _boot  # noqa: F401

from grace.server.handlers import session_status_handler
from grace.server.middleware import with_http
from grace.server.wsgi import wsgi_for

app = wsgi_for(with_http(session_status_handler))
